src/mpid/ch3/include/mpidpre.h

```diff
@@ -713,7 +713,7 @@ int MPID_Win_sync(MPIR_Win *win);
 void MPID_Progress_start(MPID_Progress_state * state);
 int MPID_Progress_wait(MPID_Progress_state * state);
 void MPID_Progress_end(MPID_Progress_state * stae);
-int MPID_Progress_test(void);
+static inline int MPID_Progress_test(void);
```

Comment:
1. This doesn't belong in this commit.
2. Is this definition needed or can it just be removed?

Reply:
It builds even without it. So, I am removing it.